lib/jellyfish/room.ex

```diff
@@ -365,6 +366,37 @@ defmodule Jellyfish.Room do
     {:noreply, state}
   end
 
+  @impl true
+  def handle_info(%Message.EndpointMessage{} = msg, state) do
+    Logger.debug("Received msg from endpoint: #{inspect(msg)}")
+    {:noreply, state}
+  end
+
+  @impl true
+  def handle_info(
+        %EndpointRemoved{endpoint_id: endpoint_id},
+        state
+      )
+      when not is_map_key(state.components, endpoint_id) or
+             not is_map_key(state.peers, endpoint_id) do
```

Comment:
I'm not a big fan of this solution, but if we're keeping it:

1. per De Morgan, I think there should be an `and` there
2. how about putting it in a `defguardp` or something? we have the same condition twice, one time it's negated